bundles/alpha.model/src/alpha/model/transformation/RaiseDependence.xtend

```diff
@@ -382,4 +418,44 @@ class RaiseDependence extends AbstractAlphaCompleteVisitor {
 		// Finally, update the context domains for the entire subtree rooted at the wrapping dependence.
 		AlphaInternalStateConstructor.recomputeContextDomain(wrappingDependence)
 	}
+	
```

Comment:
To match the formatting of the other rules, could you add the following comment block above `outReduceExpression()`?

```
	////////////////////////////////////////////////////////////
	// Reduce Expressions
	////////////////////////////////////////////////////////////
```

Reply:
Fixed.